src/System.Runtime.WindowsRuntime.UI.Xaml/pkg/build/net45/System.Runtime.WindowsRuntime.UI.Xaml.targets

```diff
@@ -0,0 +1,18 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Choose>
+    <When Condition="'$(DoNotReferenceWinRT)' != 'true'">
+      <PropertyGroup>
+        <!-- include the reference to System.Runtime -->
+        <_HasReferenceToSystemRuntime>true</_HasReferenceToSystemRuntime>
+      </PropertyGroup>
+      <ItemGroup>
+        <ReferencePath Include="$(MSBuildThisFileDirectory)..\..\ref\netstandard1.0\System.Runtime.WindowsRuntime.UI.Xaml.dll">
```

Comment:
It's probably worth explaining that we use ReferencePath so as to bypass ResolveAssemblyReference which would try to write bindingRedirects for this even though it is in the unification table.  As a side-effect of bypassing RAR we need to communicate that this will require facades to be referenced, thus above properties.

Reply:
Added